src/tests/ftest/erasurecode/restart.py

```diff
@@ -54,8 +54,10 @@ def execution(self, agg_check=None):
         self.get_dmg_command().system_start()
 
         if agg_check == "After":
+            size_after_restart = self.pool.pool_percentage_used()
+            self.log.info("Size after Restart {}".format(self.pool.pool_percentage_used()))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions